### ISSUE TITLE
Trending, Convert any image into ascii characters using pillow library in python

### ISSUE THREAD

**Issue by KanizoRGB** [section-engineering-education/engineering-education#6784]
### Topic Suggestion
Trending, Convert any image into ASCII characters using pillow library in python.

### Writing sample(s): 

1. https://www.section.io/engineering-education/the-complete-roadmap-to-blockchain-development/
2. https://www.section.io/engineering-education/an-introduction-to-machine-learning-using-c++/

#### Proposed title of article
[Languages]Trending, Convert any image into ASCII characters using pillow library in python.

#### Proposed article introduction
ASCII art is a form of art that uses the computer to present images using 95  printable characters out of the 128 characters of the ASCII standard from 1963 and ASCII compliant character set with proprietary extended characters which add to the 128 characters of the standard 7 bit ASCII. In layman's terms, it can be said to be art presented in form of text. The good thing about ASCII art is that it can be done using any text editor.
ASCII art was initially used to present images since the computers of the early days lacked the graphics ability to do so. It was also used in emails to express oneself since images could not be embedded in emails back then. However, as the world becomes nostalgic, evidenced by the remake of formerly loved films and the retro dressings in fashion, a trend has emerged of people wanting their images displayed in ASCII art.
In this tutorial, you will thus learn the code needed to convert any image into ASCII characters using the pillow library in python.

#### Key takeaways 
1. The reader will learn the use of libraries in python in solving problems
2. Tinkering with image properties to achieve any desired effect.
3. Appreciate the importance of machine learning in making work easier
4. The history and development of the ASCII standards

#### Article quality
My article is unique in the sense that it follows a step-by-step procedure explaining in simple language what is happening in each block of code. It follows an anecdotal approach of how I broke down the problem question and tried solving the problem in different ways until I found a solution.

#### References

1. https://en.wikipedia.org/wiki/ASCII_art

Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by hectorkambow** [comment]
@KanizoRGB 

Good afternoon and thank you for submitting your topic to the EngEd program. 

After some careful consideration it struck us that this topic may be a bit over saturated throughout other blog sites and official documentations.
